ci/define_pvs.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: "Define Pvs"
+  hosts: controller
+  vars:
+   base_dir: "{{ ansible_user_dir }}"
+   namespace: "service-telemetry" 
+  tasks:
+    - name: "Log into the cluster"
+      ansible.builtin.import_role:
+        name: rhol_crc
+        tasks_from: add_crc_creds.yml
+
+    - name: "Check if the install_yamls already exists"
```

Comment:
Best practice with zuul is to never clone anything in your jobs.

You can configure zuul to clone the required repos as part of the job setup.
This is done in .zuul.yaml in the `required-projects` section of the job (in this case, the parent `stf-base`)
Once a project is cloned, it is available at `{{ ansible_env.HOME }}/{{ zuul.projects[<fully qualified project name>].src_dir }}`
You can see examples of this in `ci/vars-zuul-common.yml`.

The projects you can include are listed in rdo/config repo [2].
In this case, the relevant section is in [3] and the fully-qualified project name is `github.com/openstack-k8s-operators/install_yamls`, which will also be used in `.zuul.yaml` to tell zuul to include that project.

[1] https://review.rdoproject.org/zuul/build/62079c915ab846b99f1cffc83a4b97f9/log/zuul-info/inventory.yaml
[2] https://review.rdoproject.org/cgit/config/tree/zuul/rdo.yaml
[3] https://review.rdoproject.org/cgit/config/tree/zuul/rdo.yaml#n1231

Reply:
I can't comment on an unchanged line, the required project should be added after line 14 in `.zuul.yaml`.

By default, zuul will try to check out the same branch name (i.e. master), but install_yamls does not have this branch, so you'll also need the `override-checkout` as well as the `name` for `install_yamls`.

https://zuul-ci.org/docs/zuul/latest/config/job.html#attr-job.required-projects